### PR TITLE
Cloud changelog - March 16

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated Safari recommended minimum version to v14.1+.
+
+### Improvements
+
+#### User Interface (UI)
+ - The support email field was from "Customization" to "Notifications" in the System Console. Also, the support email is now required to be set when configuring email notifications.
+ - The ping endpoint can now receive a device ID, which will report whether the device is able to receive push notifications.
+ - Added a loading indicator to the Threads global list each time more posts are fetched on infinite scroll.
+ - Added a search guidance to the Threads global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and then reach the end of the list.
+ - Added support for inline editing of posts.
```
Comment (marianunez) [inline source/install/cloud-changelog.md:26]: Reminder that this requires a feature flag to be enabled in production in Split.io

**Review by marianunez — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+## Release 2022-03-16
+
+### Compatibility
+ - Updated Safari recommended minimum version to v14.1+.
+
+### Improvements
+
+#### User Interface (UI)
+ - The support email field was from "Customization" to "Notifications" in the System Console. Also, the support email is now required to be set when configuring email notifications.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:22]:
```suggestion
 - The support email field has moved from **Customization** to **Notifications** in the System Console. Also, a support email is now required when configuring email notifications.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - The ping endpoint can now receive a device ID, which will report whether the device is able to receive push notifications.
+ - Added a loading indicator to the Threads global list each time more posts are fetched on infinite scroll.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:24]:
```suggestion
 - Added a loading indicator to the **Threads** global list each time more posts are fetched on infinite scroll.
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a search guidance to the Threads global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and then reach the end of the list.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:25]:
```suggestion
 - Added search guidance to the **Threads** global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and reach the end of the list.
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Virtualized the emoji picker and added other performance improvements to the emoji picker.
+ - Improved the performance of storing users in webapp.
+ - Fixed a small memory leak in the System Console.
+
+#### Bug Fixes
+ - Fixed a scan error on column name "LastRootPostAt": converting NULL to int64.
+ - Fixed an issue where selecting a custom status from Recent statuses used the original expiration time.
+ - Fixed an issue that caused a gap to appear on the left-hand side in products using the team sidebar.
+ - Fixed an issue where moving up or down in the channel switcher didn’t work as expected when Global Threads was in the background.
+ - Fixed an issue where clicking Enter opened the onboarding tutorial tip.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:47]:
```suggestion
 - Fixed an issue where pressing ENTER opened the onboarding tutorial tip.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where some permission checkboxes had been moved to different categories in the System Console.
+ - Fixed an issue where a blank screen occurred upon leaving a currently open unread channel with the channel unread grouping enabled.
+ - Fixed an issue related to disabling and re-enabling Custom Terms of Service.
+ - Fixed an issue where hannel links on hover overlapped the channels menus.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:51]:
```suggestion
 - Fixed an issue where channel links on hover overlapped the channels menus.
```

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed the positioning of the post menu in mobile web view.
+ - Fixed an issue where closing the keyboard shortcut modal by "ctrl/cmd + /" didn’t work.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:53]:
```suggestion
 - Fixed an issue where closing the keyboard shortcut modal by "CTRL/CMD + /" didn’t work.
```

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline. Thanks!

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (esethna) [reply source/install/cloud-changelog.md:26]: @amyblais are you taking care of that? I'll open the tix to remove the feature flag for the april on-prem release

Comment (amyblais) [reply source/install/cloud-changelog.md:26]: @esethna I've let Michel and Jelena know that the feature flag can be enabled for Cloud test servers https://community.mattermost.com/core/pl/hrpjugbec3dtzx9f4zx57nohrr.

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**